python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -195,6 +195,46 @@ def _vertex_prop_dataframe(self):
     def _edge_prop_dataframe(self):
         return self.__edge_prop_dataframe
 
+    def get_num_vertices(self, type=None):
+        """Return the number of vertices of a given type.
+
+        If type is None, return the total number of vertices.
+
+        Vertex types are set by using the `type_name` argument in
+        `add_vertex_data`.
+
+        See Also
+        --------
+        PropertyGraph.num_vertices
+        PropertyGraph.get_num_edges
+        """
+        if type is None:
+            return self.num_vertices
+        if self.__vertex_prop_dataframe is None:
+            return 0
```

Comment:
this should call             return self.num_vertices

Reply:
I don't think so.  My understanding is that vertices only referred to in the edge table don't have a type associated with them.  It would be surprising and probably incorrect to use edge type as vertex type.  So, if there is no vertex property table, then there are no typed vertices, hence 0.